cos-gpu-installer-docker/entrypoint.sh

```diff
@@ -213,9 +217,9 @@ download_kernel_src() {
     if ! download_kernel_src_from_gcs && ! download_kernel_src_from_git_repo; then
         return ${RETCODE_ERROR}
     fi
+    pushd "${KERNEL_SRC_DIR}"
+    tar xf "${COS_KERNEL_SRC_ARCHIVE}"
   fi
-  pushd "${KERNEL_SRC_DIR}"
-  tar xf "${COS_KERNEL_SRC_ARCHIVE}"
   popd
```

Comment:
This should also move into the if block :)

Reply:
Ah thanks for the catch!